site/content/en/blog/releases/1.23.0.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "1.23.0 - Kubernetes 1.22, sdk arm64 images and important bug fixes "
```

Comment:
Can you change to "1.23.0 - Kubernetes 1.22 and initial arm support"

(I don't think "bug fixes" is headline worthy since we have other things to call out)